src/metabase/lib/schema.cljc

```diff
@@ -132,13 +136,13 @@
           (aggregation-ref-errors-for-stage stage)))
 
 (defn- expression-ref-error-for-stage [stage]
-  (when-let [err-loc (first (expression-ref-errors-for-stage stage))]
+  (when-let [err-loc (some-> (expression-ref-errors-for-stage stage) first)]
```

Comment:
I don't understand how this is different or more efficient.

Reply:
clojure.lang.RT/first has a terrible implementation that tries to seq its argument. 

However, avoiding the call to `(seq nil)` is a micro-optimization indeed, it doesn't save much, so I might as well undo this change.

Reply:
TIL about that overhead, good to keep in mind.